src/snowflake/snowpark/modin/pandas/__init__.py

```diff
@@ -89,6 +88,7 @@
     )
 
 # TODO: SNOW-851745 make sure add all Snowpark pandas API general functions
+from snowflake.snowpark.modin.pandas.index import Index  # isort:skip
```

Comment:
why we want to import index her instead of later places e.g., after series, i.e., line 146?

Reply:
it gives a circular import issue if index is not imported before dataframe and series